npm/pkg/dataplane/policies/policy_windows.go

```diff
@@ -0,0 +1,58 @@
+package policies
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/Microsoft/hcsshim/hcn"
+)
+
+func convertToAclSettings(acl ACLPolicy) (hcn.AclPolicySetting, error) {
+	policySettings := hcn.AclPolicySetting{}
+	for _, setInfo := range acl.SrcList {
+		if !setInfo.Included {
+			return policySettings, fmt.Errorf("Windows Dataplane does not support negative matches. ACL: %+v", acl)
+		}
+	}
+
+	return policySettings, nil
+}
+
+func getHCNDirection(direction Direction) hcn.DirectionType {
+	switch direction {
+	case Ingress:
+		return hcn.DirectionTypeIn
+	case Egress:
+		return hcn.DirectionTypeOut
+	case Both:
```

Comment:
seems this case is not necessary.

Reply:
Windows dataplane does accept rules on both directions, in case of network policies with both ingress and egress rules i can use a DENY all with BOTH, that way we get away with single rule than 2

Reply:
I meant `case Both` is not necessary based on logic. Anyway, by default it returns ""